mathcomp/Makefile.test-suite.coq.local

```diff
@@ -0,0 +1,20 @@
+OUTPUT_TESTS=test_suite/output.v
+
+OUTPUT_ARTIFACTS=$(OUTPUT_TESTS:%.v=%.v.out.new)
+
+COQ_VERSION=$(shell $(COQC) -print-version | cut -d ' ' -f 1 | cut -d '.' -f 1-2)
+
+# Given a file f we compare its compilation output f.out.new with
+# f.out.COQ_VERSION (or f.out if f.out.COQ_VERSION does not exist)
+post-all:: $(OUTPUT_ARTIFACTS)
+	@for f in $(OUTPUT_TESTS); do\
+	  if [ -e $$f.out.$(COQ_VERSION) ]; then REFERENCE=$$f.out.$(COQ_VERSION);\
```

Comment:
@gares 
I've just found the origin of issue 2. mentioned in https://github.com/math-comp/math-comp/pull/433#issuecomment-691033191 (variable name clash)

```suggestion
COQ_VERSION_MINOR=$(shell $(COQC) -print-version | cut -d ' ' -f 1 | cut -d '.' -f 1-2)

# Given a file f we compare its compilation output f.out.new with
# f.out.COQ_VERSION_MINOR (or f.out if f.out.COQ_VERSION_MINOR does not exist)
post-all:: $(OUTPUT_ARTIFACTS)
	@for f in $(OUTPUT_TESTS); do\
	  if [ -e $$f.out.$(COQ_VERSION_MINOR) ]; then REFERENCE=$$f.out.$(COQ_VERSION_MINOR);\
```